code/network/multiui.cpp

```diff
@@ -2734,7 +2734,8 @@ void multi_sg_init_gamenet()
 	
 	Net_player->tracker_player_id = Multi_tracker_id;
 
-	Multi_sg_netgame->security = (rand() % 32766) + 1;			// get some random security number	
+	// DISCUSSME: do we want security to remain within such a narrow range? Out of scope?
+	Multi_sg_netgame->security = util::Random::next(32766) + 1;			// get some random security number	
```

Comment:
Since this is a straight port of the old functionality this is fine as it is. We could increase the range in a separate PR which would also make that change more independent.

Reply:
Yup agreed that the behavior here shouldn't change in this PR. I can open an issue to discuss whether we should expand the range of valid values for `Multi_sg_netgame->security `.

Reply:
Opened Issue #3336 to discuss this question.